framework/platform/bpl/uci/cfg/bpl_cfg_uci.cpp

```diff
@@ -20,6 +20,28 @@ extern "C" {
 
 #define PRINTF(LEVEL, fmt, args...) printf(LEVEL ":{%s, %d}:" fmt, __func__, __LINE__, ##args)
 
+#ifdef BEEROCKS_RDKB
```

Comment:
Please add a comment here and in the commit message saying that this is a temp solution which will be replaced by Adam's PR - and add a link to this PR.

Reply:
Fixed. But I didn't link to Adams PR since currently, it does not exist.